core/src/mindustry/graphics/OverlayRenderer.java

```diff
@@ -107,15 +107,20 @@ public void drawTop(){
         Lines.stroke(buildFade * 2f);
 
         if(buildFade > 0.005f){
+            float absin = 0.5f + Mathf.absin(Time.time, 10f, 0.5f);
             state.teams.eachEnemyCore(player.team(), core -> {
                 float dst = core.dst(player);
                 if(dst < state.rules.enemyCoreBuildRadius * 2.2f){
                     Draw.color(Color.darkGray);
                     Lines.circle(core.x, core.y - 2, state.rules.enemyCoreBuildRadius);
-                    Draw.color(Pal.accent, core.team.color, 0.5f + Mathf.absin(Time.time, 10f, 0.5f));
+                    Draw.color(Pal.accent, core.team.color, adsin);
                     Lines.circle(core.x, core.y, state.rules.enemyCoreBuildRadius);
                 }
             });
+
+            Draw.color(Pal.accent);
+            Draw.alpha(adsin);
+            Lines.circle(player.x, player.y, 208.2f);
```

Comment:
what is `208.2f`? is that the build radius for (all?) units? why is it hardcoded in here?

Reply:
I asked Anuke in what code moment this saved
But Anuken don't told me

Reply:
https://github.com/Anuken/Mindustry/blob/5654e3120dfa0398e1199a1748c38b6e6b63208e/core/src/mindustry/Vars.java#L86

Reply:
Thx